app/main.py

```diff
@@ -1,34 +1,42 @@
 from http import HTTPStatus
 from typing import Annotated
 
-from fastapi import Depends, FastAPI, File, HTTPException, Request, UploadFile, status
+from fastapi import Depends, FastAPI, Form, HTTPException, Request, UploadFile
+from fastapi.concurrency import asynccontextmanager
 from fastapi.responses import JSONResponse
-from pipeline import GPT, OCR, FertilizerInspection
+from pipeline import GPT, OCR
+from psycopg_pool import ConnectionPool
 from pydantic import UUID4
 
-from app.config import lifespan
-from app.connection_manager import ConnectionManager
+from app.config import Settings, configure
 from app.controllers.data_extraction import extract_data
-from app.controllers.inspections import read, read_all
-from app.controllers.users import sign_in, sign_up
+from app.controllers.inspections import create, read, read_all
+from app.controllers.users import sign_up
 from app.dependencies import (
     authenticate_user,
     fetch_user,
-    get_connection_manager,
+    get_connection_pool,
     get_gpt,
     get_ocr,
+    get_settings,
+    validate_files,
 )
-from app.exceptions import (
-    InspectionNotFoundError,
-    UserConflictError,
-    UserNotFoundError,
-    log_error,
-)
+from app.exceptions import InspectionNotFoundError, UserConflictError, log_error
```

Comment:
It would probably help us later on to import the entire controller. Sometimes different controllers might have the same method names.

import app.controller.inspections as inspections

inspections.create(...)

easier to tell at a glance which domain is handling an api route

Thoughts?

Reply:
method names could also use some verbosity

create_record
read_one_record
read_all_records

Reply:
Done